From 039629540f1a6f26ee5a7eabd26832ca81f7996f Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Thu, 17 Jan 2008 21:27:50 +0000
Subject: [PATCH] MSys' gcc does not like -mwin32

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 W11/w32/Makefile  |   12 ++++++------
 W11/wrap/Makefile |    2 +-
 2 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/W11/w32/Makefile b/W11/w32/Makefile
index 1a82bbb..13972a8 100644
--- a/W11/w32/Makefile
+++ b/W11/w32/Makefile
@@ -11,18 +11,18 @@ alldoc tags:
 	dllwrap --export-all-symbols -o ../lib/libW11.dll xlib.o ntutil.o event.o -mwindows
 
 xlib.o: xlib.c ntdef.h
-	gcc -c xlib.c ${CFLAGS} -I.. -mwin32
+	gcc -c xlib.c ${CFLAGS} -I..
 
 ntutil.o: ntutil.c ntdef.h
-	gcc -c ntutil.c ${CFLAGS} -I.. -mwin32
+	gcc -c ntutil.c ${CFLAGS} -I..
 
 event.o: event.c ntdef.h
-	gcc -c event.c ${CFLAGS} -I.. -mwin32
+	gcc -c event.c ${CFLAGS} -I..
 
 debug:
-	gcc -c xlib.c -g -I.. -mwin32
-	gcc -c event.c -g -I.. -mwin32
-	gcc -c ntutil.c -g -I.. -mwin32
+	gcc -c xlib.c -g -I..
+	gcc -c event.c -g -I..
+	gcc -c ntutil.c -g -I..
 
 clean:
 	-rm ../lib/libW11.dll xlib.o ntutil.o event.o
diff --git a/W11/wrap/Makefile b/W11/wrap/Makefile
index f79d22e..a13e35f 100644
--- a/W11/wrap/Makefile
+++ b/W11/wrap/Makefile
@@ -11,7 +11,7 @@ alldoc tags:
 	ar r ../lib/libX11.a wrap.o rxvt_res.o
 
 wrap.o: wrap.c xwrappers.gen
-	gcc ${CFLAGS} -c wrap.c -I.. -mwin32
+	gcc ${CFLAGS} -c wrap.c -I..
 
 rxvt_res.o: rxvt.rc
 	windres -i rxvt.rc -o rxvt_res.o
-- 
1.5.3.6.2073.ga1e8

