From 0ee9877ec32f69b73dfd3c20340f633c7dd88976 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Thu, 17 Jan 2008 21:28:43 +0000
Subject: [PATCH] MSysify

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 src/ptytty.c |    2 +-
 src/screen.c |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/ptytty.c b/src/ptytty.c
index 71ca4b1..d5475fd 100644
--- a/src/ptytty.c
+++ b/src/ptytty.c
@@ -46,7 +46,7 @@
 #ifdef HAVE_SYS_IOCTL_H
 # include <sys/ioctl.h>
 #endif
-#if defined(PTYS_ARE_PTMX) && !defined(__CYGWIN32__)
+#if defined(PTYS_ARE_PTMX) && !defined(__CYGWIN32__) && !defined(__MSYS__)
 # include <sys/stropts.h>      /* for I_PUSH */
 #endif
 
diff --git a/src/screen.c b/src/screen.c
index 44053b7..891f814 100644
--- a/src/screen.c
+++ b/src/screen.c
@@ -2805,7 +2805,7 @@ rxvt_selection_property(rxvt_t *r, Window win, Atom prop)
 	    XFree(s);
 	if (a != Success)
 	    return;
-#ifndef __CYGWIN32__
+#ifndef __MSYS__
 	if (atype == r->h->xa[XA_INCR]) {	/* start an INCR transfer */
 	    D_SELECT((stderr, "rxvt_selection_property: INCR: starting transfer"));
 	    XDeleteProperty(r->Xdisplay, win, prop);
-- 
1.5.3.6.2073.ga1e8

