From 62938fb9d28644323544f45fb55788c5465a44d1 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Mon, 26 Nov 2012 17:03:50 -0600
Subject: [PATCH 3/3] Add a .gitignore file

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 .gitignore | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)
 create mode 100644 .gitignore

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..cb76a07
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1,16 @@
+*.[oa]
+*.pyc
+/build/
+/Makefile
+/Makefile.pre
+/Misc/python.pc
+/Modules/Setup.config
+/Modules/Setup.local
+/Modules/config.c
+/Modules/ld_so_aix
+/Parser/pgen.exe
+/Parser/pgen.stamp
+/config.log
+/config.status
+/python.exe
+/pyconfig.h
-- 
1.8.0.msysgit.0.1.g1cbcfae

