From b84ac332247aed6142e386cfef05fad79ff1250c Mon Sep 17 00:00:00 2001
From: KernelJ <kernelj@epixxware.com>
Date: Mon, 19 Jan 2015 02:59:24 +0000
Subject: [PATCH 1/2] Change machine type to i686 for mingw32

---
 conftools/config.sub | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/conftools/config.sub b/conftools/config.sub
index 30fdca8..a690448 100644
--- a/conftools/config.sub
+++ b/conftools/config.sub
@@ -773,7 +773,7 @@ case $basic_machine in
 		basic_machine=microblaze-xilinx
 		;;
 	mingw32)
-		basic_machine=i386-pc
+		basic_machine=i686-pc
 		os=-mingw32
 		;;
 	mingw32ce)
-- 
2.1.0

