From 25fdc40f66defe1a4a2a7e4cf4556981cb00baf5 Mon Sep 17 00:00:00 2001
From: Christian Stimming <stimming@tuhh.de>
Date: Tue, 22 Jan 2008 23:42:42 +0100
Subject: [PATCH] Fix aprutils makefile

---
 apr-util/Makefile.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/apr-util/Makefile.in b/apr-util/Makefile.in
index 1700892..a02fe2b 100644
--- a/apr-util/Makefile.in
+++ b/apr-util/Makefile.in
@@ -81,7 +81,7 @@ install: $(TARGET_LIB)
 
 $(TARGET_LIB):
 	@objects="`find $(SUBDIRS) -name expat -prune -o -name 'gen_uri_delims.@so_ext@' -prune -o -name '*.@so_ext@' -print`"; \
-	    tmpcmd="$(LINK) @lib_target@ @EXTRA_OS_LINK@"; \
+	    tmpcmd="$(LINK) @lib_target@ $(APRUTIL_LIBS) @EXTRA_OS_LINK@"; \
 	    echo $$tmpcmd; \
 	    $$tmpcmd && touch $@
 
-- 
1.5.4-rc3.GIT

