From 33e72e24d000cececb5e6a95dd68cfd03e487d5b Mon Sep 17 00:00:00 2001
From: Christian Stimming <stimming@tuhh.de>
Date: Tue, 22 Jan 2008 23:43:03 +0100
Subject: [PATCH] Fix mingw detection during configure

---
 configure.in |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.in b/configure.in
index f5a22ce..b36e891 100644
--- a/configure.in
+++ b/configure.in
@@ -185,7 +185,7 @@ dnl It should always work but with libtool 1.4.3 on OS X it breaks the build.
 dnl So we only turn it on for platforms where we know we really need it.
 AC_MSG_CHECKING([whether libtool needs -no-undefined])
 case $host in
-  *-*-cygwin*)
+  *-*-mingw32*)
     AC_MSG_RESULT([yes])
     LT_NO_UNDEFINED="-no-undefined"
     ;;
@@ -198,7 +198,7 @@ AC_SUBST(LT_NO_UNDEFINED)
 
 AC_MSG_CHECKING([whether to avoid circular linkage at all costs])
 case $host in
-  *-*-cygwin*)
+  *-*-mingw32*)
     AC_MSG_RESULT([yes])
     AC_DEFINE([SVN_AVOID_CIRCULAR_LINKAGE_AT_ALL_COSTS_HACK], 1,
               [Define if circular linkage is not possible on this platform.])
-- 
1.5.4-rc3.GIT

