From f4df26d484c8a210cf59d48ac768f0796055fda7 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Sat, 25 Jul 2009 17:11:34 +0200
Subject: [PATCH] IOV_MAX is defined too large on MSys; ignore it

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 serf/context.c |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/serf/context.c b/serf/context.c
index 9516fb7..5fc1dc3 100644
--- a/serf/context.c
+++ b/serf/context.c
@@ -26,6 +26,11 @@
    ### stop, rebuild a pollset, and repopulate it. what suckage.  */
 #define MAX_CONN 16
 
+/* MSys defines IOV_MAX too large */
+#ifdef __MSYS__
+#undef IOV_MAX
+#endif
+
 /* Windows does not define IOV_MAX, so we need to ensure it is defined. */
 #ifndef IOV_MAX
 #define IOV_MAX 16
-- 
1.6.3.msysgit.0.14.gb078c

