From 63ccb9e736f6d7184d36907a65813981f86065a2 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Sat, 25 Jul 2009 18:56:56 +0200
Subject: [PATCH] Add a .gitignore file

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 .gitignore |   41 +++++++++++++++++++++++++++++++++++++++++
 1 files changed, 41 insertions(+), 0 deletions(-)
 create mode 100644 .gitignore

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..7bd4c46
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1,41 @@
+*.o
+*.lo
+*.a
+*.la
+*.pc
+*.dll
+*.exe
+*.mk
+.make.dirs
+*-config
+pkginfo
+.libs
+.deps
+Makefile
+config.log
+config.nice
+config.status
+/apr-util/export_vars.sh
+/apr-util/include/apr_ldap.h
+/apr-util/include/apu.h
+/apr-util/include/apu_want.h
+/apr-util/include/private/apu_config.h
+/apr-util/include/private/apu_select_dbm.h
+/apr-util/xml/expat/config.h
+/apr-util/xml/expat/lib/expat.h
+/apr-util/xml/expat/libtool
+/apr-util/xml/expat/libtool.bak
+/apr/include/apr.h
+/apr/include/arch/unix/apr_private.h
+/apr/libtool
+/apr/strings/apr_snprintf.loT
+/contrib/client-side/svn_load_dirs/svn_load_dirs.pl
+/contrib/client-side/svncopy/svncopy.pl
+/contrib/client-side/svncopy/testsvncopy.pl
+/contrib/hook-scripts/commit-email.pl
+/neon/config.h
+/neon/test/makekeys
+/subversion/bindings/swig/perl/native/Makefile.PL
+/subversion/svn_private_config.h
+/tools/backup/hot-backup.py
+/tools/hook-scripts/commit-access-control.pl
-- 
1.6.3.msysgit.0.14.gb078c

