From 7a4b2d9d2a9ade8074f2ed06c8ee470da5719bc2 Mon Sep 17 00:00:00 2001
From: Laurent Boulard <laurent.boulard@gmail.com>
Date: Tue, 28 Jul 2009 23:05:20 +0200
Subject: [PATCH] Makes sqlite3.c to compile and work with MSYS dll.

---
 sqlite-amalgamation/sqlite3.c |    7 +++++++
 1 files changed, 7 insertions(+), 0 deletions(-)

diff --git a/sqlite-amalgamation/sqlite3.c b/sqlite-amalgamation/sqlite3.c
index 6206770..daee3b7 100644
--- a/sqlite-amalgamation/sqlite3.c
+++ b/sqlite-amalgamation/sqlite3.c
@@ -19,6 +19,12 @@
 **
 ** This amalgamation was generated on 2009-04-15 12:45:14 UTC.
 */
+
+#ifdef __MSYS__
+#define SQLITE_OS_UNIX 1
+#define SQLITE_DISABLE_LFS 1
+#endif
+
 #define SQLITE_CORE 1
 #define SQLITE_AMALGAMATION 1
 #ifndef SQLITE_PRIVATE
--
1.6.3.msysgit.0.424.g8520d

