From 804bf50773b8cbd63d5208e950f2f04d540c63ff Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Wed, 15 Jul 2009 03:39:48 +0200
Subject: [PATCH 1/7] Cope better with DOS line endings

We need to strip the CRs explicitely, it seems.

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>

diff --git a/util/mk1mf.pl b/util/mk1mf.pl
index 0ddd2ca..5460ed1 100755
--- a/util/mk1mf.pl
+++ b/util/mk1mf.pl
@@ -289,6 +289,7 @@ for (;;)
 	{
 	chop;
 
+	$_ =~ s/\r$//;
 	($key,$val)=/^([^=]+)=(.*)/;
 	if ($key eq "RELATIVE_DIRECTORY")
 		{
-- 
2.1.0

