From 4a7aa374eb02bef807bdacf031ad1af081633040 Mon Sep 17 00:00:00 2001
From: Thomas Braun <thomas.braun@virtuell-zuhause.de>
Date: Wed, 13 Apr 2011 10:34:04 +0200
Subject: [PATCH] do not specify march i486

Signed-off-by: Thomas Braun <thomas.braun@virtuell-zuhause.de>
---
 Configure |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Configure b/Configure
index 429ab2e..8bf69d4 100755
--- a/Configure
+++ b/Configure
@@ -505,7 +505,7 @@ my %table=(
 "BC-32","bcc32::::WIN32::BN_LLONG DES_PTR RC4_INDEX EXPORT_VAR_AS_FN:${no_asm}:win32",
 
 # MinGW
-"mingw", "gcc:-mno-cygwin -DL_ENDIAN -DWIN32_LEAN_AND_MEAN -fomit-frame-pointer -O3 -march=i486 -Wall::-D_MT:MINGW32:-lws2_32 -lgdi32 -lcrypt32:BN_LLONG ${x86_gcc_des} ${x86_gcc_opts} EXPORT_VAR_AS_FN:${x86_asm}:coff:win32:cygwin-shared:-D_WINDLL -DOPENSSL_USE_APPLINK:-mno-cygwin:.dll.a",
+"mingw", "gcc:-mno-cygwin -DL_ENDIAN -DWIN32_LEAN_AND_MEAN -fomit-frame-pointer -O3 -Wall::-D_MT:MINGW32:-lws2_32 -lgdi32 -lcrypt32:BN_LLONG ${x86_gcc_des} ${x86_gcc_opts} EXPORT_VAR_AS_FN:${x86_asm}:coff:win32:cygwin-shared:-D_WINDLL -DOPENSSL_USE_APPLINK:-mno-cygwin:.dll.a",
 # As for OPENSSL_USE_APPLINK. Applink makes it possible to use .dll
 # compiled with one compiler with application compiled with another
 # compiler. It's possible to engage Applink support in mingw64 build,
-- 
1.7.4.msysgit.0.915.gacc77

