From 370c821fc36d7dde48e9572cb77aa27517ad0111 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Wed, 15 Jul 2009 03:47:52 +0200
Subject: [PATCH 4/4] Install to /mingw/

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Makefile b/Makefile
index 57d742e..85994d7 100644
--- a/Makefile
+++ b/Makefile
@@ -26,7 +26,7 @@ HERE=.
 # for, say, /usr/ and yet have everything installed to /tmp/somedir/usr/.
 # Normally it is left empty.
 INSTALL_PREFIX=
-INSTALLTOP=/usr/local/ssl
+INSTALLTOP=/mingw
 
 # Do not edit this manually. Use Configure --openssldir=DIR do change this!
 OPENSSLDIR=/usr/local/ssl
-- 
1.6.3.2.1299.gee46c

