From 5eaed2b8a6c2b9917c5423ac78befdec18e8cb0a Mon Sep 17 00:00:00 2001
From: Erik Faye-Lund <kusmabite@gmail.com>
Date: Mon, 27 Jul 2009 23:27:04 +0200
Subject: [PATCH] generate import lib

Signed-off-by: Erik Faye-Lund <kusmabite@gmail.com>
---
 util/pl/Mingw32.pl |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/util/pl/Mingw32.pl b/util/pl/Mingw32.pl
index 87c2ccc..10a3d05 100644
--- a/util/pl/Mingw32.pl
+++ b/util/pl/Mingw32.pl
@@ -94,7 +94,7 @@ sub do_lib_rule
 
 	$ret.="$target: \$(${Name}OBJ)\n";
 	$ret.="\tif exist $target \$(RM) $target\n";
-	$ret.="\t\$(MKLIB) $target \$(${Name}OBJ) \$(EX_LIBS) $ex\n\n";
+	$ret.="\t\$(MKLIB) $target \$(${Name}OBJ) \$(EX_LIBS) -Wl,--out-implib,$target.a $ex\n\n";
 	}
 
 sub do_link_rule
-- 
1.6.3.2

