From 4ea960d7154b1e51b24dd0fea61613b6b0574297 Mon Sep 17 00:00:00 2001
From: Erik Faye-Lund <erik.faye-lund@hue.no>
Date: Thu, 5 Jun 2014 17:19:43 +0200
Subject: [PATCH] include limits.h for INT_MAX

---
 ssl/s3_pkt.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ssl/s3_pkt.c b/ssl/s3_pkt.c
index c087fee..32a7a29 100644
--- a/ssl/s3_pkt.c
+++ b/ssl/s3_pkt.c
@@ -111,6 +111,7 @@
 
 #include <stdio.h>
 #include <errno.h>
+#include <limits.h>
 #define USE_SOCKETS
 #include "ssl_locl.h"
 #include <openssl/evp.h>
-- 
1.9.2.msysgit.0.161.g83227c1

