diff --git a/configure b/configure
index bd9edd2..23a9d60 100644
--- a/configure
+++ b/configure
@@ -119,9 +119,9 @@ if test "$gcc" -eq 1 && ($cc -c $cflags $test.c) 2>/dev/null; then
   fi
   case "$uname" in
   Linux* | linux* | GNU | GNU/* | *BSD | DragonFly) LDSHARED=${LDSHARED-"$cc -shared -Wl,-soname,libz.so.1,--version-script,zlib.map"} ;;
-  CYGWIN* | Cygwin* | cygwin* | OS/2*)
+  CYGWIN* | Cygwin* | cygwin* | OS/2* | MINGW*)
         EXE='.exe' ;;
-  MINGW*|mingw*)
+  DISABLED_MINGW*|mingw*)
 # temporary bypass
         rm -f $test.[co] $test $test$shared_ext
         echo "Please use win32/Makefile.gcc instead."
