From 95898bfa20d9e10bcdf789d869652770714d2351 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Sat, 25 Jul 2009 17:44:54 +0200
Subject: [PATCH] Add a .gitignore file

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 .gitignore |   18 ++++++++++++++++++
 1 files changed, 18 insertions(+), 0 deletions(-)
 create mode 100644 .gitignore

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..42de541
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1,18 @@
+*.o
+*.lo
+*.la
+*.spec
+*.list
+*.pc
+/lib/ca-bundle.crt
+/lib/config.h
+/libtool
+/src/config.h
+/src/curl.exe
+stamp-h*
+Makefile
+.deps
+.libs
+/config.log
+/config.status
+/curl-config
-- 
1.6.3.2.1299.gee46c

