From d861b5be20777f6c28ee30d0d4aaca416e3174e2 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Sat, 25 Jul 2009 17:44:54 +0200
Subject: [PATCH 2/3] Add a .gitignore file

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..67aaa4d
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1,18 @@
+*.o
+*.lo
+*.la
+*.spec
+*.list
+*.pc
+/lib/ca-bundle.crt
+/lib/config.h
+/libtool
+/src/config.h
+/src/curl.exe
+stamp-h*
+Makefile
+.deps
+.libs
+/config.log
+/config.status
+/curl-config
-- 
2.1.0

