From 93031ceedbc29eaa742ef91a31acdcd0fa01a739 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Fri, 9 Nov 2007 22:08:30 +0000
Subject: [PATCH] Fix compilation on msysGit

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 MSYS-STUFF/msys-build |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/MSYS-STUFF/msys-build b/MSYS-STUFF/msys-build
index 0f5ec70..d10c7d9 100644
--- a/MSYS-STUFF/msys-build
+++ b/MSYS-STUFF/msys-build
@@ -31,6 +31,7 @@ echo "manually fixup config.sh, config.h, myconfig in aother window?"
 echo -n "hit return to continue: "
 read dummy
 
+export PERL5LIB="`pwd`"/lib
 make 2>&1 | tee MSYS-STUFF/log-make.txt
 
 
@@ -48,7 +49,6 @@ echo "    op/taint"
 echo -n "press enter to continue"
 read dummy
 
-export PERL5LIB=/usr/local/src/perl/perl-5.6.1-MSYS-1.0.11/lib
 make test 2>&1 | tee MSYS-STUFF/log-test.txt
 (cd t;./perl harness) 2>&1 | tee MSYS-STUFF/log-harness.txt
 
-- 
1.5.3.5.2233.g50e2

