From f569241dd2d27051adfb7df4dd56348e6a8b4766 Mon Sep 17 00:00:00 2001
From: Simon Sasburg <Simon.Sasburg@gmail.com>
Date: Sun, 11 Nov 2007 12:26:27 +0100
Subject: [PATCH] Fix compiling if you have paths with spaces in your $PATH.

---
 Makefile.SH |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/Makefile.SH b/Makefile.SH
index df357ea..9815331 100644
--- a/Makefile.SH
+++ b/Makefile.SH
@@ -153,6 +153,8 @@ EOT
 	;;
 esac
 
+ldlibpth=`echo $ldlibpth\" | sed 's/=/="/'`
+
 : Prepare dependency lists for Makefile.
 dynamic_list=' '
 for f in $dynamic_ext; do
-- 
1.5.3.5.2233.g50e2

