From e2f55d27c054e36ad5979edf133c3f23cff40279 Mon Sep 17 00:00:00 2001
From: Simon Sasburg <Simon.Sasburg@gmail.com>
Date: Fri, 5 Oct 2007 22:35:45 +0200
Subject: [PATCH] fix issue with trying to link with -lm

---
 ext/POSIX/Makefile.PL |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/ext/POSIX/Makefile.PL b/ext/POSIX/Makefile.PL
index 9cd24b0..8d690fd 100644
--- a/ext/POSIX/Makefile.PL
+++ b/ext/POSIX/Makefile.PL
@@ -7,7 +7,7 @@ use ExtUtils::Constant 0.11 'WriteConstants';
 use Config;
 my @libs;
 if ($^O ne 'MSWin32') {
-    @libs = ('LIBS' => ["-lm -lposix -lcposix"]);
+    @libs = ('LIBS' => ["-lposix -lcposix"]);
 }
 WriteMakefile(
     NAME	=> 'POSIX',
-- 
1.5.3.5.2233.g50e2

