From f146120c0dec8da685cb751a15b18af1a89da6cb Mon Sep 17 00:00:00 2001
From: Simon Sasburg <Simon.Sasburg@gmail.com>
Date: Wed, 26 Sep 2007 18:31:38 +0200
Subject: [PATCH] Make 'install' a phony target so it still works in the presence of the 'INSTALL' file

---
 Makefile.SH |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/Makefile.SH b/Makefile.SH
index 3872706..88c2242 100644
--- a/Makefile.SH
+++ b/Makefile.SH
@@ -572,6 +572,7 @@ extra.pods: miniperl
 install-strip:
 	$(MAKE) STRIPFLAGS=-s install
 
+.PHONY: install
 install:
 	$(MAKE) install.perl install.man STRIPFLAGS=$(STRIPFLAGS)
 
-- 
1.5.3.729.gd231

