From d60450116d146e6f0f4e375429b910ba370ac828 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Fri, 21 Dec 2007 05:53:54 +0100
Subject: [PATCH] MallocCfg_ptr is really EXT, not extern

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 malloc_ctl.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/malloc_ctl.h b/malloc_ctl.h
index 7a6aed0..5d877ab 100644
--- a/malloc_ctl.h
+++ b/malloc_ctl.h
@@ -55,7 +55,7 @@ enum {
   MallocCfgP_last
 };
 START_EXTERN_C
-extern IV *MallocCfg_ptr;
+EXT IV *MallocCfg_ptr;
 extern char **MallocCfgP_ptr;
 END_EXTERN_C
 
-- 
1.5.3.5.2233.g50e2

