From 29f6a242845459e8c4b1b84ccae07f1178aafb2c Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Fri, 21 Dec 2007 06:40:53 +0100
Subject: [PATCH] Add a .gitignore file

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 .gitignore |   13 +++++++++++++
 1 files changed, 13 insertions(+), 0 deletions(-)
 create mode 100644 .gitignore

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..4dcec06
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1,13 @@
+*.o
+*.a
+*.dll
+*.bs
+*.exh
+*.fnm
+blib
+pm_to_blib
+/Policy.sh
+/config.h
+/config.sh
+/GNUmakefile
+Makefile
-- 
1.5.3.5.2233.g50e2

