From 7e3feef625f353a7520a12ff5cb7483c8e91310b Mon Sep 17 00:00:00 2001
From: Sebastian Schuberth <sschuberth@gmail.com>
Date: Tue, 29 Dec 2009 01:22:06 +0100
Subject: [PATCH] Include MinGW compatibility header

Signed-off-by: Sebastian Schuberth <sschuberth@gmail.com>
---
 poptconfig.c |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/poptconfig.c b/poptconfig.c
index ce88751..9fd228b 100644
--- a/poptconfig.c
+++ b/poptconfig.c
@@ -10,6 +10,9 @@
 #include "poptint.h"
 #include <sys/stat.h>
 
+#include <dirent.h>
+#include "compat/mingw.h"
+
 #if defined(HAVE_FNMATCH_H)
 #include <fnmatch.h>
 
-- 
1.6.5.rc2.13.g1be2

