From 2895d372a4660422ca274ae9f0268b6a46ab8f74 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Mon, 7 Dec 2009 17:20:14 +0100
Subject: [PATCH 1/6] Do not include headers on Windows that do not exist there

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 configure.ac   |    1 +
 src/compress.c |    2 ++
 2 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/configure.ac b/configure.ac
index f13b364..53942b8 100644
--- a/configure.ac
+++ b/configure.ac
@@ -69,6 +69,7 @@ AC_CHECK_HEADERS(stdint.h fcntl.h locale.h stdint.h inttypes.h unistd.h)
 AC_CHECK_HEADERS(utime.h wchar.h wctype.h limits.h)
 AC_CHECK_HEADERS(getopt.h err.h)
 AC_CHECK_HEADERS(sys/mman.h sys/stat.h sys/types.h sys/utime.h sys/time.h)
+AC_CHECK_HEADERS(sys/ioctl.h)
 AC_CHECK_HEADERS(zlib.h)
 
 dnl Checks for typedefs, structures, and compiler characteristics.
diff --git a/src/compress.c b/src/compress.c
index f04ab27..10b2158 100644
--- a/src/compress.c
+++ b/src/compress.c
@@ -45,7 +45,9 @@ FILE_RCSID("@(#)$File: compress.c,v 1.64 2009/05/08 17:41:58 christos Exp $")
 #endif
 #include <string.h>
 #include <errno.h>
+#ifdef HAVE_SYS_IOCTL_H
 #include <sys/ioctl.h>
+#endif
 #ifdef HAVE_SYS_WAIT_H
 #include <sys/wait.h>
 #endif
-- 
1.7.3

