From c2b06e879ce72169f80cf046f4d80cbb002e6b33 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Mon, 7 Dec 2009 17:21:21 +0100
Subject: [PATCH 2/6] On msysGit, include pthread.h for struct timespec

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 src/cdf_time.c |    4 ++++
 src/readcdf.c  |    4 ++++
 2 files changed, 8 insertions(+), 0 deletions(-)

diff --git a/src/cdf_time.c b/src/cdf_time.c
index 14dcfc6..65a7a99 100644
--- a/src/cdf_time.c
+++ b/src/cdf_time.c
@@ -31,6 +31,10 @@ FILE_RCSID("@(#)$File: cdf_time.c,v 1.8 2009/06/20 20:47:30 christos Exp $")
 #endif
 
 #include <time.h>
+#ifdef WIN32
+/* definition of timespec */
+#include <pthread.h>
+#endif
 #ifdef TEST
 #include <err.h>
 #endif
diff --git a/src/readcdf.c b/src/readcdf.c
index 52cf579..c81eacd 100644
--- a/src/readcdf.c
+++ b/src/readcdf.c
@@ -33,6 +33,10 @@ FILE_RCSID("@(#)$File: readcdf.c,v 1.22 2010/01/20 01:36:55 christos Exp $")
 #include <unistd.h>
 #include <string.h>
 #include <time.h>
+#ifdef WIN32
+/* definition of timespec */
+#include <pthread.h>
+#endif
 #include <ctype.h>
 
 #include "cdf.h"
-- 
1.7.3

