From 3e1b29eb1dda508ee78b02591b927f3fc03679d6 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Mon, 7 Dec 2009 17:22:32 +0100
Subject: [PATCH 4/7] Avoid using ELOOP on Windows

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 src/fsmagic.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/fsmagic.c b/src/fsmagic.c
index 537fb14..a848f15 100644
--- a/src/fsmagic.c
+++ b/src/fsmagic.c
@@ -70,9 +70,11 @@ bad_link(struct magic_set *ms, int err, char *buf)
 	    == -1)
 		return -1;
 	else if (!mime) {
+#ifndef WIN32
 		if (err == ELOOP)
 			errfmt = "symbolic link in a loop";
 		else
+#endif
 			errfmt = "broken symbolic link to `%s'";
 		if (ms->flags & MAGIC_ERROR) {
 			file_error(ms, err, errfmt, buf);
-- 
1.9.5.msysgit.0.4.gceda007

