From 291ac13faaf652dfd33ab6393cf054208510203c Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Mon, 7 Dec 2009 17:22:32 +0100
Subject: [PATCH 4/6] Avoid using ELOOP on Windows

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 src/fsmagic.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/src/fsmagic.c b/src/fsmagic.c
index a68cd10..d53b916 100644
--- a/src/fsmagic.c
+++ b/src/fsmagic.c
@@ -70,9 +70,11 @@ bad_link(struct magic_set *ms, int err, char *buf)
 	    == -1)
 		return -1;
 	else if (!mime) {
+#ifndef WIN32
 		if (err == ELOOP)
 			errfmt = "symbolic link in a loop";
 		else
+#endif
 			errfmt = "broken symbolic link to `%s'";
 		if (ms->flags & MAGIC_ERROR) {
 			file_error(ms, err, errfmt, buf);
-- 
1.6.5.1.1371.g1abf3

