From 1b971dcbe6e5981c64f458f7f3bbc7eb7839057c Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Mon, 7 Dec 2009 17:23:24 +0100
Subject: [PATCH 6/6] On msysGit, include the whole regex.c

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 src/softmagic.c |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/src/softmagic.c b/src/softmagic.c
index 88f5214..294ef70 100644
--- a/src/softmagic.c
+++ b/src/softmagic.c
@@ -41,6 +41,9 @@ FILE_RCSID("@(#)$File: softmagic.c,v 1.135 2009/03/27 22:42:49 christos Exp $")
 #include <stdlib.h>
 #include <time.h>
 
+/* hack for msysGit */
+#include <regex.c>
+
 
 private int match(struct magic_set *, struct magic *, uint32_t,
     const unsigned char *, size_t, int);
-- 
1.6.5.1.1371.g1abf3

