From f69d49282f0bafa601a53e29797989c9960f61a7 Mon Sep 17 00:00:00 2001
From: Steffen Prohaska <prohaska@zib.de>
Date: Sun, 4 May 2008 14:02:45 +0200
Subject: [PATCH] Tweak msys_p2w for msysgit

---
 src/winsup/cygwin/path.cc |    8 ++++++--
 1 files changed, 6 insertions(+), 2 deletions(-)

diff --git a/source/winsup/cygwin/path.cc b/source/winsup/cygwin/path.cc
index e701360..582ea77 100644
--- a/source/winsup/cygwin/path.cc
+++ b/source/winsup/cygwin/path.cc
@@ -3134,10 +3134,13 @@ msys_p2w (char const * const path)
       //
       // Prevent http://some.string/ from being modified.
       // 
-      if ((sspath > 0 && strlen (sspath) > 2)
+      if (((sspath > 0 && strlen (sspath) > 2)
 	  && (sspath[1] == '/')
 	  && (sspath[2] == '/')
 	  )
+	  // Prevent git's :/message syntax from beeing modified.
+	  || (strlen (spath) > 1 && spath[0] == ':' && spath[1] == '/')
+	  )
 	{
 	  debug_printf("returning: %s", path);
 	  return ((char *)path);
@@ -3153,7 +3156,8 @@ msys_p2w (char const * const path)
 	   // 
 	   // Prevent ``foo:echo /bar/baz'' from being considered a path list.
 	   // 
-	   && (strlen (sspath) > 1 && strchr (":./", sspath[1]) > 0)
+	   // Prevent :: from beeing considered a path list [msysgit].
+	   && (strlen (sspath) > 1 && strchr ("./", sspath[1]) > 0)
 	   )
     {
       //
-- 
1.5.5.1025.g3502

