From 446c37bf8f4e08dfca20fedee8de3f5dfe1b32ef Mon Sep 17 00:00:00 2001
From: Steffen Prohaska <prohaska@zib.de>
Date: Sat, 10 May 2008 20:30:09 +0200
Subject: [PATCH] p2w for msysgit: Check if string starts with "." or "/" before considering it a pathlist

---
 src/winsup/cygwin/path.cc |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/source/winsup/cygwin/path.cc b/source/winsup/cygwin/path.cc
index 59827e3..8d73b09 100644
--- a/source/winsup/cygwin/path.cc
+++ b/source/winsup/cygwin/path.cc
@@ -3157,6 +3157,8 @@ msys_p2w (char const * const path)
 	   // remember that this is a recursive routine.
 	   // 
 	   && (strchr ("-\"\'", spath[0]) == 0)
+	   // Prevent modification if string does not start with "." or "/".
+	   && (strchr ("./", spath[0]) > 0)
 	   // 
 	   // Prevent ``foo:echo /bar/baz'' from being considered a path list.
 	   // 
-- 
1.5.5.1015.g9d258

