From 5f10e96b8fe8330c012369a53a987e12ed947f37 Mon Sep 17 00:00:00 2001
From: Steffen Prohaska <prohaska@zib.de>
Date: Sat, 10 May 2008 20:30:09 +0200
Subject: [PATCH] p2w for msysgit: Check if string starts with "." or "/" before considering it a pathlist

---
 src/winsup/cygwin/path.cc |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/src/winsup/cygwin/path.cc b/src/winsup/cygwin/path.cc
index 59827e3..7969b6e 100644
--- a/src/winsup/cygwin/path.cc
+++ b/src/winsup/cygwin/path.cc
@@ -3144,6 +3144,8 @@ msys_p2w (char const * const path)
 	  )
 	  // Prevent git's :/message syntax from beeing modified.
 	  || (strlen (spath) > 1 && spath[0] == ':' && spath[1] == '/')
+	  // Prevent modification if string does not start with "." or "/".
+	  || (strchr ("./", spath[0]) == 0)
 	  )
 	{
 	  debug_printf("returning: %s", path);
-- 
1.5.5.1015.g9d258

