From 33bad2b11e2f14ea2d03835c00606d69b4f37f6c Mon Sep 17 00:00:00 2001
From: Simon Sasburg <Simon.Sasburg@gmail.com>
Date: Wed, 26 Sep 2007 18:31:38 +0200
Subject: [PATCH] Make 'install' a phony target so it still works in the presence of the 'INSTALL' file

---
 Makefile.SH |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/Makefile.SH b/Makefile.SH
index ac5ade4..608e5d0 100644
--- a/Makefile.SH
+++ b/Makefile.SH
@@ -570,6 +570,7 @@ extra.pods: miniperl
 install-strip:
 	$(MAKE) STRIPFLAGS=-s install
 
+.PHONY: install
 install:
 	$(MAKE) install.perl install.man STRIPFLAGS=$(STRIPFLAGS)
 
-- 
1.5.3.GIT

