From 6519a860abd42262b92641f2911c44a9a1df5ec9 Mon Sep 17 00:00:00 2001
From: Simon Sasburg <Simon.Sasburg@gmail.com>
Date: Sun, 11 Nov 2007 12:26:27 +0100
Subject: [PATCH] Fix compiling if you have paths with spaces in your $PATH.

---
 Makefile.SH |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/Makefile.SH b/Makefile.SH
index 9f02306..3872706 100644
--- a/Makefile.SH
+++ b/Makefile.SH
@@ -103,6 +103,8 @@ true)
 	;;
 esac
 
+ldlibpth=`echo $ldlibpth\" | sed 's/=/="/'`
+
 : Prepare dependency lists for Makefile.
 dynamic_list=' '
 for f in $dynamic_ext; do
-- 
1.5.3.729.gd231

