From 7501423f40832f8894f785165aac0ce6b6d06557 Mon Sep 17 00:00:00 2001
From: Simon Sasburg <Simon.Sasburg@gmail.com>
Date: Fri, 5 Oct 2007 22:35:45 +0200
Subject: [PATCH] fix issue with trying to link with -lm

---
 ext/POSIX/Makefile.PL |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/ext/POSIX/Makefile.PL b/ext/POSIX/Makefile.PL
index 73bb02d..d82c738 100644
--- a/ext/POSIX/Makefile.PL
+++ b/ext/POSIX/Makefile.PL
@@ -2,7 +2,7 @@ use ExtUtils::MakeMaker;
 use Config;
 my @libs;
 if ($^O ne 'MSWin32') {
-    @libs = ('LIBS' => ["-lm -lposix -lcposix"]);
+    @libs = ('LIBS' => ["-lposix -lcposix"]);
 }
 WriteMakefile(
     NAME	=> 'POSIX',
-- 
1.5.3.729.gd231

