From aaa5e6e398e1e4dc527df0e95c5c07e32118cb79 Mon Sep 17 00:00:00 2001
From: Erik Faye-Lund <kusmabite@gmail.com>
Date: Thu, 29 Sep 2011 21:55:40 +0200
Subject: [PATCH] use windows exe-suffix

---
 Makefile.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Makefile.in b/Makefile.in
index 40906a2..590bf8f 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -81,7 +81,7 @@ G=@G@
 ZCAT=@ZCAT@
 # To install zcat executable and man page as gzcat, use: ZCAT=gzcat
 
-X=
+X=.exe
 # For OS/2 or MSDOS, use: X=.exe
 
 O=.o
-- 
1.7.6.355.g842ba.dirty

