From 3d7333df34b22b394ac6ff3253cbf9d311285558 Mon Sep 17 00:00:00 2001
From: Pat Thoyts <patthoyts@users.sourceforge.net>
Date: Sat, 2 Feb 2013 00:08:12 +0000
Subject: [PATCH] Support case-insensitive filenames for assembler processing

Signed-off-by: Pat Thoyts <patthoyts@users.sourceforge.net>
---
 mpi/Makefile.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/mpi/Makefile.in b/mpi/Makefile.in
index 54392c7..603fd11 100644
--- a/mpi/Makefile.in
+++ b/mpi/Makefile.in
@@ -641,8 +641,8 @@ uninstall-am:
 # cancel the default rules used by libtool which do not really
 # work and add one to cpp .S files
 .S.o:
-	 $(CPP) $(INCLUDES) $(DEFS) $< | grep -v '^#' > _$*.s
-	 $(COMPILE) $(AM_CCASFLAGS) -c _$*.s
+	 $(CPP) -x assembler-with-cpp $(INCLUDES) $(DEFS) $< | grep -v '^#' > _$*.asm
+	 $(COMPILE) -x assembler $(AM_CCASFLAGS) -c _$*.asm
 	 mv -f _$*.o $*.o
 
 .S.lo:
-- 
1.8.1.msysgit.1

