From 88142e78d8d5ce2fa113020d24730717fe2088f3 Mon Sep 17 00:00:00 2001
From: Erik Faye-Lund <kusmabite@gmail.com>
Date: Mon, 27 Jul 2009 19:57:23 +0200
Subject: [PATCH] fix openssl library names

---
 configure |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/configure b/configure
index c1981db..2426af8 100644
--- a/configure
+++ b/configure
@@ -24464,7 +24464,7 @@ fi
   rpathdirs=
   ltrpathdirs=
   names_already_handled=
-  names_next_round='ssl32 eay32'
+  names_next_round='ssl crypto'
   while test -n "$names_next_round"; do
     names_this_round="$names_next_round"
     names_next_round=
-- 
1.6.3.2

