From b4925358bf53ca3858c69cca9aacb8ea6b6f1c62 Mon Sep 17 00:00:00 2001
From: KernelJ <kernelj@epixxware.com>
Date: Sun, 18 Jan 2015 19:47:32 +0000
Subject: [PATCH 2/2] Include sockets before OpenSSL

---
 src/tls.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/tls.c b/src/tls.c
index 162d540..0a85000 100644
--- a/src/tls.c
+++ b/src/tls.c
@@ -33,6 +33,11 @@
 #include <time.h>
 #include <errno.h>
 
+#if W32_NATIVE
+# include <sys/socket.h>
+# include "w32sock.h"
+#endif
+
 #ifdef HAVE_LIBGNUTLS
 # include <gnutls/gnutls.h>
 # include <gnutls/x509.h>
@@ -50,11 +55,6 @@
 # include <idna.h>
 #endif
 
-#if W32_NATIVE
-# include <sys/socket.h>
-# include "w32sock.h"
-#endif
-
 #include "gettext.h"
 #include "xalloc.h"
 #include "xvasprintf.h"
-- 
2.1.0

